internal/coreinternal/processor/filterexpr/matcher.go

```diff
@@ -36,9 +36,29 @@ func (e *env) HasLabel(key string) bool {
 	return ok
 }
 
-func (e *env) Label(key string) string {
-	v, _ := e.attributes.Get(key)
-	return v.StringVal()
+func (e *env) Label(key string) interface{} {
```

Comment:
This is an exported function whose signature is changing.  Where is it used and how is that adapted to the changed signature?

Reply:
yup it is used in the expr values in configs i t now returns interface{} instead of string meaning it will always return the correct type of the value now instead of always returning a string value or "" if it could not be cast to a string value

Reply:
@Aneurysm9 this is only used in the expr package as far as I can see  no where else.